.github/workflows/build.yml

```diff
@@ -173,7 +174,7 @@ jobs:
                 issue_number: Number(issue_number),
                 owner: context.repo.owner,
                 repo: context.repo.repo,
-                labels: ['authorized-request']
+                labels: ['authorized-request', 'automerge']
```

Comment:
what is the semantic of new `automerge` label? and how is different from `sanity-ok`? this steps will be executed  under the same condition as the step in line 31: https://github.com/openshift-helm-charts/charts/pull/177/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R31 IMHO, both labels have the same semantic, Hence, I would not introduce `automerge`.  Furthermore I would even argue that `sanity-ok` label is as well not needed - `authorized-request` label is added when sanity check is successful - so they have the same semantic.

What I miss however is a label (e.g. `valid-chart-pr`) stating that chart checks are passing - and that should come as the outcome of `verify-pr` step. With that, PR should merged only if both `authorized-request` and `valid-chart-pr` labels are present.

cc @mmulholla - would like to hear your opinion.